src/core/client/stream/tabs/Comments/Comment/InReplyTo.tsx

```diff
@@ -2,21 +2,51 @@ import { Localized } from "@fluent/react/compat";
 import cn from "classnames";
 import React, { FunctionComponent } from "react";
 
+import { useCoralContext } from "coral-framework/lib/bootstrap";
+import { globalErrorReporter } from "coral-framework/lib/errors";
 import CLASSES from "coral-stream/classes";
+import computeCommentElementID from "coral-stream/tabs/Comments/Comment/computeCommentElementID";
 import { Flex, Icon } from "coral-ui/components/v2";
 
+import { CommentContainer_comment as CommentData } from "coral-stream/__generated__/CommentContainer_comment.graphql";
+
 import styles from "./InReplyTo.css";
 
 interface Props {
-  username: string;
+  parent: CommentData["parent"];
+  isLink: boolean;
```

Comment:
I suggest here as well to rename the property to what it does like before: e.g. `enableJumpToParent` or `disableJumpToParent` depending on how you want the default to be.